packages/create-rise/src/cli.ts

```diff
@@ -96,11 +124,13 @@ async function copyAdditionalTemplateFiles(root: string) {
 
 createRise().catch((e) => {
```

Comment:
Meanwhile @TheKohan while we wait for @ericvicenti thoughts on the logo, I have another quick one for you!

We should switch $`npm install` to be $({ quiet: !verbose })`npm install` and use `minimist` from `zx` to check for that flag and allow passing `--verbose`.

`npm install` generates excessive output and will make the beautiful UI of the CLI you created much uglier!